test/email-field-events.html

```diff
@@ -72,6 +73,40 @@
           });
         });
       });
+
+      describe('has-input-value-changed event with clear button', () => {
```

Comment:
Could we make it a sub-suite of the existing `has-input-value-changed event` to re-use this part:

```js
hasInputValueChangedSpy = sinon.spy();
valueChangedSpy = sinon.spy();
emailField = fixture('email-field');
emailField.addEventListener('has-input-value-changed', hasInputValueChangedSpy);
emailField.addEventListener('value-changed', valueChangedSpy);
input = emailField.inputElement;
```

```js
describe('with clear button', () => {
  beforeEach(() => {
    emailField.clearButtonVisible = true;
  });

  describe('with user input', () => {
    ...
  });
});
```